server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -247,18 +316,29 @@ export class TenantManager {
             throw new NetworkError(403, `Tenant, ${tenantId}, does not exist.`);
         }
 
+        fetchTenantKeyMetric.success(`Retrieved tenant keys from the database.`);
+
         const encryptedTenantKey1 = tenantDocument.key;
         const tenantKey1 = this.secretManager.decryptSecret(encryptedTenantKey1);
         if (tenantKey1 == null) {
             winston.error("Tenant key1 decryption failed.");
-            Lumberjack.error("Tenant key1 decryption failed.", { [BaseTelemetryProperties.tenantId]: tenantId });
+            Lumberjack.error("Tenant key1 decryption failed.", lumberProperties);
```

Comment:
Not your code, but Line 321 - 361 is a little bit sloppy here. We can do following if my understanding is right:
```
        let tenantKey1;
        let tenantKey2;

        const encryptedTenantKey1 = tenantDocument.key;
        tenantKey1 = this.secretManager.decryptSecret(encryptedTenantKey1);
        if (tenantKey1 == null) {
            winston.error("Tenant key1 decryption failed.");
            Lumberjack.error("Tenant key1 decryption failed.", lumberProperties);
            throw new NetworkError(500, "Tenant key1 decryption failed.");
        }
       const encryptedTenantKey2 = tenantDocument.secondaryKey;
        if (encryptedTenantKey2) {
            tenantKey2 = this.secretManager.decryptSecret(encryptedTenantKey2);
            if (tenantKey2 == null) {
               winston.error("Tenant key2 decryption failed.");
               Lumberjack.error("Tenant key2 decryption failed.", { [BaseTelemetryProperties.tenantId]: tenantId });
               throw new NetworkError(500, "Tenant key2 decryption failed.");
           }
       } else {
            winston.info("Tenant key2 doesn't exist.");
            Lumberjack.info("Tenant key2 doesn't exist.", lumberProperties);
       }

     const cacheKeys = {
         key1: encryptedTenantKey1,
         key2: encryptedTenantKey2,
     }
     await this.setCacheKey(tenantId, cacheKeys);
        return {
            key1: tenantKey1,
            key2: tenantKey2,
        };
      
```

Reply:
Updated this section